lib/snippets.coffee

```diff
@@ -298,6 +299,13 @@ module.exports =
   expandSnippetsUnderCursors: (editor) ->
     return false unless snippet = @snippetToExpandUnderCursor(editor)
 
+    @getStore(editor).observeHistory({
```

Comment:
We should really supply a flag in the change event indicating when the change is coming from an `undo` or `redo`. If you'd have any interest in adding that, I'd be happy to shepherd it. Then we could simplify this.

Reply:
Yeah, that’s certainly a better idea. I’ll investigate.